nestkernel/connection_manager.cpp

```diff
@@ -1122,7 +1116,13 @@ nest::ConnectionManager::get_connections( ArrayDatum& connectome,
 #ifdef _OPENMP
 #pragma omp critical( get_connections )
 #endif
-        connectome.append_move( conns_in_thread );
+        {
+          while ( !conns_in_thread.empty() )
```

Comment:
@alexeyshusharin Fine! Moving the code to a method with an informative name will make the code as readable as with `std::copy()` and save a bit of memory.